src/test/regress/sql/plpgsql_cache.sql

```diff
@@ -688,3 +688,42 @@ select cache_test(t) from cache_tab;
 
 drop function get_dummy_string(text);
 drop function cache_test(text);
+
+-- ************************************************************
+-- * Test partition elimination inside plpgsql
+-- ************************************************************
+
+drop table if exists t_part;
+create table t_part (a date, b int) distributed by (b)
+partition by range (a)
+(start ('2019-01-01') end ('2020-01-01') every (interval '1 month'));
+
+create or replace function f_pe(d date)
+returns void as $$
+begin
+	perform * from t_part where a = d;
+end;
+$$ language plpgsql
+	set optimizer = off
+	set client_min_messages = debug1
+	set debug_print_plan = on
+	set debug_pretty_print = on;
+
+drop language if exists plpythonu cascade;
```

Comment:
Wrap these commands with ignore block or don't drop language because it may cause removing depending object created by other tests:

> drop language if exists plpythonu cascade;
NOTICE:  drop cascades to function isspilling(text)
create language plpythonu;